EIPS/eip-6963.md

```diff
@@ -75,6 +75,16 @@ data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12
 
 The `icon` string MUST be a data URI as defined in [RFC-2397]. The image SHOULD be squared with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur.
 
+#### RDNS
+
+The **`rdns`** (Reverse-DNS) property serves to provide an identifier which DApps can rely on to be stable between sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions.
+The Reverse-DNS convention implies that the value should start with a reversed DNS domain name controlled by the Provider. The domain name should be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
+
+- The DNS part of the `rdns` value MUST BE a valid [RFC-1034] Domain Name;
```

Comment:
```suggestion
- The `rdns` value MUST BE a valid [RFC-1034] Domain Name;
```
I feel like exempting the "subdomain or product name" segment from RFC-1034 just invites problems, while requiring product names to be encoded as valid subdomains will be a stitch in time if future verification specs define garden-variety `/well-known/` lookups :D

Reply:
Here I used the wording "DNS part of the `rdns`" only to mean the original non-reversed string.
We can't say that the literal `rnds` value needs to be a valid domain, because "com.example" isn't. It's the reversed-to-the original string that needs to be a valid domain :) 
I'm not sure how to say this without making the sentence unnecessarily long 

I agree that there's no need to exempt the "product name" part from being a valid part of the domain rfc.

At the same time I feel we have to highlight that, given a value of `com.example.wallet-name`, the following is expected:
* `example.com` SHOULD be an _active_ domain
* `wallet-name.example.com` MUST BE a _valid domain name_,
* `wallet-name.example.com` is NOT required to be an _active_ domain.

Perhaps this is better done with an example?